templates/fields.html.twig

```diff
@@ -138,22 +138,30 @@
                             'display_emptychoice' : false,
                             'action'              : 'get_items_from_itemtype',
                             'dom_rand'            : rand,
+                            'width'               : '100%',
                         }
                 ]) %}
 
-                <span id='results_items_id{{rand}}' class='form-field row col-12 col-sm-6  mb-2'>
-                    {% if value.itemtype != '' %}
-                        {{ macros.dropdownField(value.itemtype, items_id_prefix ~ name, value.items_id, label, field_options|merge({
-                            'entity': value.itemtype == 'User' ? -1 : item.getEntityID(),
-                            'rand': rand,
-                            'right': 'all',
-                            'display_emptychoice' : false,
-                            'no_label': true
-                        })) }}
-                    {% endif %}
-                </span>
-            {% endif %}
-        </div>
+                {# fake label for DOM disposition #}
+                <label class="col-form-label col-xxl-4 text-xxl-end" for="dropdown_items_id_testfield{{rand}}">
+                </label>
+
+                <div class="col-xxl-8 field-container">
+                    <span id='results_items_id{{rand}}' class='col-lg-14'>
+                        {% if value.itemtype != '' %}
+                            {{ macros.dropdownField(value.itemtype, items_id_prefix ~ name, value.items_id, ' ', field_options|merge({
+                                'entity': value.itemtype == 'User' ? -1 : item.getEntityID(),
+                                'rand': rand,
+                                'right': 'all',
+                                'displaywith': ['otherserial', 'serial'],
```

Comment:
I've noticed a difference in display between ```updateItemOnSelectEvent``` dropdown and ```macros.dropdownField``` dropdown.

Because ```updateItemOnSelectEvent``` call ```dropdownAllItems``` which defaults to ```['otherserial', 'serial']```